WebExample/playwright.config.ts

```diff
@@ -0,0 +1,29 @@
+import {defineConfig, devices} from '@playwright/test';
+import * as TEST_CONST from '../testConstants';
+
+export default defineConfig({
+  testDir: './tests',
+  preserveOutput: 'never',
+  outputDir: undefined,
```

Comment:
Do we really need to set this prop? What's the default value?

Reply:
the default is `<package.json-directory>/test-results`, as long as we don't use reports I don't see any point in using it + if i remember correctly there was an issue in CI when trying to preserve artifacts from tests.